packages/terra-profile-image/src/ProfileImage.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import TerraImage from 'terra-image';
+import avatarImage from './avatar.svg';
+import './ProfileImage.scss';
+
+/* eslint react/no-unused-prop-types: [0] */
+const propTypes = {
+  /**
+   * The source for the image which will be displayed.
+   */
+  src: PropTypes.string.isRequired,
+  /**
+   * Sets the style of the image from the following values; `default`, `rounded`, `circle`, `thumbnail`.
+   */
+  variant: PropTypes.oneOf(['default', 'rounded', 'circle', 'thumbnail']),
+  /**
+   * Sets the fluid behavior of the image, which is `nonfluid` by default.
+   */
+  isFluid: PropTypes.bool,
+  /**
+   * The text content that specifies an alternative text for an image.
+   */
+  alt: PropTypes.string.isRequired,
+  /**
+   * Sets the height of the image.
+   */
+  height: PropTypes.string,
+  /**
+   * Sets the width of the image.
+   */
+  width: PropTypes.string,
+  /**
+   * Function to be executed when the profile image load is successful.
+   */
+  onLoad: PropTypes.func,
+  /**
+   * Function to be executed when the profile image load errors.
+   */
+  onError: PropTypes.func,
+};
+
+const defaultProps = {
+  variant: 'default',
+  isFluid: false,
+  alt: ' ',
```

Comment:
In the general sense, I think it's okay to mark it as required and provide a default value.  Basically it's just preventing someone from trying to set the value to null.  However, I'm not sure there is much value in defaulting the 'alt' prop to a blank space.  I copied this over from terra-image.  Does anyone know why 'alt' is required but defaulted to blank space?  Seems like we could just not require the field.  @aganapuram @bjankord @Matt-Butler